decidim/decidim
# Redesign: remove images from documents section when removing them from modal

#### :tophat: What? Why?
Remove images from the documents section when removing them from the modal and avoid the cancel button to remove items from the modal as it was causing a weird behavior if you followed the steps below:
1. Add an image in the modal and click on "Next".
2. Open the modal again and click on "Cancel".
3. Open the modal again and the not-removed image disappears, so if you click on "Next", it disappears from the documents section.
 
#### :pushpin: Related Issues
- Fixes #11347 

#### Testing
1. Edit the initiative
2. Attach 2 images (like select, attach, click next)
3. Open the edit pop-up, click "remove" (on one image)
4. Then press "cancel"
5. See the image has not been removed
6. Open the edit pop-up (see the previously removed image is still there)

### :camera: Screenshots
https://github.com/decidim/decidim/assets/6973564/76513667-d247-4485-8ede-c16b30e18d88

:hearts: Thank you!
